File: net/Link.Message.Client/content/AudioMessageContent.cs
namespace Link.Message.Client.content
{

	/// <summary>
	/// 音频消息内容
	/// </summary>
	public class AudioMessageContent : FileMessageContent
	{

		public AudioMessageContent(string downloadUrl, int size, string fileName, string extension) : base(downloadUrl, size, fileName, extension)
		{
		}

		public AudioMessageContent(string downloadUrl, int size, string fullFileName) : base(downloadUrl, size, fullFileName)
		{
		}
	}

}

File: java/link-message-client/src/main/java/link/message/client/content/AudioMessageContent.java
package link.message.client.content;

/**
 * 音频消息内容
 * @author zhongt
 *
 */
public class AudioMessageContent extends FileMessageContent {
	
	public AudioMessageContent(String downloadUrl, int size, String fileName, String extension) {
		super(downloadUrl, size, fileName, extension);
	}
	
	public AudioMessageContent(String downloadUrl, int size, String fullFileName) {
		super(downloadUrl, size, fullFileName);
	}
}
